jupyter-docker/Dockerfile

```diff
@@ -1,34 +1,30 @@
 # adapted from https://hub.docker.com/r/jupyter/base-notebook/ AKA https://github.com/jupyter/docker-stacks/tree/master/base-notebook
 
-# Debian Jessie debootstrap from 2017-02-27
-# https://github.com/docker-library/official-images/commit/aa5973d0c918c70c035ec0746b8acaec3a4d7777
-FROM debian@sha256:52af198afd8c264f1035206ca66a5c48e602afb32dc912ebf9e9478134601ec4
+FROM debian:jessie
 
 USER root
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV DEBIAN_REPO http://cdn-fastly.deb.debian.org
 
-ENV JAVA_VER jdk1.8.0_131
-ENV JAVA_TGZ jdk-8u131-linux-x64.tar.gz
-ENV JAVA_URL http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/$JAVA_TGZ
+ENV JAVA_VER jdk1.8.0_161
+ENV JAVA_TGZ jdk-8u161-linux-x64.tar.gz
+ENV JAVA_URL http://download.oracle.com/otn-pub/java/jdk/8u161-b12/2f38c3b165be4555a1fa6e98c45e0808/$JAVA_TGZ
 ENV JAVA_HOME /usr/lib/jdk/$JAVA_VER
 
+# mount this from the outside
+# note Dataproc supports Spark 2.2.0, but there are no pre-packaged Hail distributions past Spark 2.1.0. See:
```

Comment:
Add a comment matching Dataproc 1.1 to Spark 2.0.2, like in the Leo code.

Reply:
👍